lz4-sys/src/lib.rs

```diff
@@ -80,6 +80,7 @@ pub struct LZ4FFrameInfo {
     pub block_size_id: BlockSize,
     pub block_mode: BlockMode,
     pub content_checksum_flag: ContentChecksum,
+    pub content_size: c_ulonglong,
     pub reserved: [c_uint; 5],
```

Comment:
I believe the size of the `reserved` field must be changed to match. (My C/FFI skills are not sufficient to know what its size should be, but see https://github.com/lz4/lz4/blob/77d1b93f72628af7bbde0243b4bba9205c3138d9/lib/lz4frame.h#L175-L183 for the original definition)